docs/source/index.rst

```diff
@@ -1,6 +1,15 @@
 Welcome to developer documentation for Leapp!
-=================================
-   `Leapp <https://oamg.github.io/leapp/>`_ is an OS and application modernization framework
+=============================================
+`Leapp <https://oamg.github.io/leapp/>`_ is an OS and application modernization framework.
+
+Resources not included in this RTD:
+ - RPM dependencies for `leapp-repository <https://github.com/oamg/leapp-repository/blob/master/docs/dependencies.md>`_.
+ - Upgrade inhibition by producing a `inhibiting report message <https://github.com/oamg/leapp-repository/blob/master/docs/howto-inhibit-upgrade.md>`_.
```

Comment:
This should be turned into proper tutorial

Reply:
I agree with @jmikovic , contents from the both links should be moved into RTD. I don't see the reason why they are buried inside leapp-repository.